### DISCUSSION TITLE
Connect table1 and table2

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#12015] [Help]
Hey,

I have used Multiple Variable ,but page show the chosen have  2 table in 1 screen. When showing Preview It only shows one table.

Table1:
![image](https://github.com/Budibase/budibase/assets/93240913/31a45f94-24f8-4738-884e-b0a346b8b1e7)
||
V 
![image](https://github.com/Budibase/budibase/assets/93240913/192d980c-83e1-4a69-b934-cffefc6641c2)

Table2:
![image](https://github.com/Budibase/budibase/assets/93240913/6b51b706-1b23-4686-ad6d-8cf75331a257)
||
V 
![image](https://github.com/Budibase/budibase/assets/93240913/ef2e7f04-4463-4068-8b5a-19739122d070)

I want both tables to be displayed.

Thank you,

**Comment by ConorWebb96** [comment]
Hey @LTCTHITA,

What version of Budibase are you running?
What data source are you using? E.g. externalDB (MySQL etc) or internal the internal budibase db?

Can you provide some screenshots of your component tree? Can you also provide a screenshot of your tables within the data screen that you are using here?

Do you have any strange filters on these that may be preventing one or the other table from displaying when the other has data in it?

Thanks in advance for the additional information.

**Reply by melohagan** [reply]
To add to this one, are URL variables being used here? Make sure they are correct - the preview may not populate this with any value which could result in no rows being returned.

- https://docs.budibase.com/docs/url-parameters
- https://github.com/Budibase/budibase/issues/7502
